modules/mod_ginger_rdf/models/m_rdf.erl

```diff
@@ -103,14 +103,25 @@ ensure_resource(Uri, Props, Context) ->
 %% @doc Create non-authoritative RDF resource
 -spec create_resource(string(), list(), #context{}) -> integer().
 create_resource(Uri, Props, Context) ->
-    AllProps = [
+    AdminContext = z_acl:sudo(Context),
```

Comment:
Can we do without this? I’d like to keep the original user as it provides some information about who originally created the RDF resource.

Reply:
Why do we care about who made the resource initially? I think that's information that belongs on the edge not the resource in this case. The "creator" of a non authoritative resources is not the user but an external author.